server/controller/index.js

```diff
@@ -0,0 +1,9 @@
+const express = require('express');
+const login = require('./login');
+
+const router = express.Router();
+
+router.post('/api/login', login);
```

Comment:
a more clear way of writing the router would be
```suggestion
router
.route('/api/login')
.post(login);
```